src/+FeatureCreators/FeatureSet5NSrcDetection.m

```diff
@@ -0,0 +1,366 @@
+            % duet
+            % wSTFTSec;   % window size in seconds for STFT
+            % wDUETSec;   % window size in seconds for DUET histogram
+            % hDUETSec;   % window shift for duet historgram
+            % binsAlpha;  % number of histogram bins for alpha dimension
+            % binsDelta;  % number of histogram bins for delta dimension
+            % maxAlpha;   % masking threshold for alpha dimension
+            % maxDelta;   % masking threshold for delta dimension
+            afeRequests{1}.name = 'duet';
+            afeRequests{1}.params = genParStruct( ...
+                'duet_wSTFTSec', obj.wSizeSec,...
+                'duet_wDUETSec', (1/2),...
+                'duet_hDUETSec', (1/6));
+            
+            % internaural level differences
+            afeRequests{2}.name = 'ild';
+            afeRequests{2}.params = genParStruct(...
+                commonParams{:}, ...
+                'fb_nChannels', obj.nFreqChannels);
+            
+            % interaural time differences
+            afeRequests{3}.name = 'itd'; % same as segmentationKs
+            afeRequests{3}.params = genParStruct( ...
+                commonParams{:}, ...
+                'fb_nChannels', obj.nFreqChannels);
+            
+            % onset strengths
+            afeRequests{4}.name = 'onsetStrength';
+            afeRequests{4}.params = genParStruct( ...
+                commonParams{:}, ...
+                'fb_nChannels', obj.nFreqChannels);
+
+            % spectral features
+            afeRequests{5}.name = 'spectralFeatures';
+            afeRequests{5}.params = genParStruct( ...
+                commonParams{:}, ...
+                'fb_nChannels', obj.nFreqChannels ...
+                );
+
+            % cross correlation
+            afeRequests{6}.name = 'crosscorrelation';
+            afeRequests{6}.params = genParStruct( ...
+                commonParams{:}, ...
+                'fb_nChannels', obj.nFreqChannels ...
+                );
+        end
+        
+        function x = constructVector( obj )
+            % constructVector from afe requests
+            %   #1: DUET, #2: ILD, #3: ITD, #4: OnS, #5: Spectral
+            %
+            %   See getAFErequests
+            
+            x = {};
+            
+            % afeIdx 1: STFT -> DUET histogram + summary
+            duet = obj.afeData(1);
+            if ~isa(duet, 'cell')
+                duet = {duet};
+            end
+            duet_hist = obj.makeDuetFeatureBlock(...
+                duet{1},...
+                ~obj.descriptionBuilt,...
+                obj.duetIntegrate);
+            duet_hist = obj.reshapeBlock(duet_hist, 1);
+            duet_hist = obj.reshape2featVec(duet_hist);
+            x = obj.concatFeats(x, duet_hist);
+            % duet histogram summary
+            duet_summ = obj.makePeakSummaryFeatureBlock(...
+                duet_hist{1},...
```

Comment:
Not sure whether this is intended: DUET features are scanned for peaks row-wise (1-dimensional instead of 2-dimensional), because they are transformed to a vector in line 105...

@pmeier82 ?